core/src/main/java/io/lionweb/lioncore/java/serialization/JsonSerialization.java

```diff
@@ -235,68 +243,71 @@ private void considerLanguageDuringSerialization(
     }
   }
 
-  public SerializedChunk serializeNodesToSerializationBlock(Node... nodes) {
+  public <T extends ClassifierInstance<?>> SerializedChunk serializeNodesToSerializationBlock(
```

Comment:
Couldn't we just take `ClassifierInstance<*>... classifierInstances`? This would remove the need for the type parameter